pkg/imager/ova/ova.go

```diff
@@ -249,7 +249,7 @@ func renderOVF(name string, size, capacity int64) (string, error) {
 	}{
 		VMDK:     name + ".vmdk",
 		Size:     size,
-		Capacity: capacity,
+		Capacity: capacity / (1 << 20),
```

Comment:
should we set this in `capacity` variable then, or is only applicable to OVA,? I guess `Capacity` field expects a MB value?

Reply:
it's just the way we build the XML template... we say in the XML that it's in MiB. I could change the template, but I'm less sure it will work :-D

Reply:
I will probably rename the argument to `diskSize` to match other usecases

Reply:
yeh, just a comment saying it expects MiB is fine for future reference

Reply:
renamed to make things less confusing